src/app/page.tsx

```diff
@@ -6,9 +6,10 @@ import Images from "./images";
 import CroppingResizingViewSettings from "./cropping-resizing-view-settings";
 import Navigation from "./navigation";
 import Response_and_correction from "./response_and_correction";
+import Progress from "./progress";
 
 const DEBUG = true;
-const fakePipeline = false;
+const fakePipeline = true;
```

Comment:
This is making the progress messages work incorrectly, but removing this, everything seems good to me